src/main/java/com/thealgorithms/others/Sudoku.java

```diff
@@ -40,22 +51,31 @@ public static boolean isSafe(int[][] board, int row, int col, int num) {
             }
         }
 
-        // if there is no clash, it's safe
         return true;
     }
 
+    /**
+     * Solves the Sudoku puzzle using backtracking.
+     * The algorithm finds an empty cell and tries placing numbers from 1 to 9.
```

Comment:
Maybe we should also point out here that 1...9 is standard, but it could be for 1...n too

Reply:
Okay, I added the following comment to this function documentation:
`The standard version of Sudoku uses numbers from 1 to 9, so the algorithm can be easily modified for other variations of the game.`

Reply:
And we need to have this here:

`Solves the Sudoku puzzle using backtracking. The algorithm finds an empty cell and tries placing numbers from 1 to n, where n is the size of the board (for example, from 1 to 9 in a standard 9x9 Sudoku).`

Reply:
Changed the function docstring to include the above statement.